src/H5Zscaleoffset.c

```diff
@@ -1221,6 +1224,8 @@ H5Z__filter_scaleoffset(unsigned flags, size_t cd_nelmts, const unsigned cd_valu
         minval_size = sizeof(unsigned long long) <= ((unsigned char *)*buf)[4] ? sizeof(unsigned long long)
                                                                                : ((unsigned char *)*buf)[4];
         minval      = 0;
+        if (H5_IS_BUFFER_OVERFLOW((unsigned char *)*buf, 5 + minval_size, (unsigned char*)*buf + *buf_size - 1))
```

Comment:
```suggestion
        if (H5_IS_BUFFER_OVERFLOW((unsigned char *)*buf, 5 + minval_size,
                                                        (unsigned char *)*buf + *buf_size - 1))
```
clang-format changes that should let the clang-format check pass